generator/src/main/java/com/scottlogic/deg/generator/walker/reductive/ReductiveIterationVisualiser.java

```diff
@@ -12,15 +14,21 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.FileSystems;
 import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.nio.file.spi.FileSystemProvider;
 import java.util.concurrent.atomic.AtomicInteger;
 
 public class ReductiveIterationVisualiser implements IterationVisualiser {
     private final AtomicInteger currentIteration = new AtomicInteger();
     private final Path visualiseDirectoryPath;
 
-    public ReductiveIterationVisualiser(Path outputDirectory) {
-        this.visualiseDirectoryPath = outputDirectory.resolve("reductive-walker");
+    @Inject
+    public ReductiveIterationVisualiser(@Named("outputPath") Path outputPath) {
+        Path directoryPath = outputPath.getParent() == null
+            ? Paths.get(System.getProperty("user.dir"))
```

Comment:
Does this mean that if someone doesn't specify an output path this will start creating files in unexpected places?

Reply:
You have to specify this path for violations, I believe

Reply:
Also there is no current way to enable this mode from the command line.